bindings/python/crocoddyl/multibody/frames.cpp

```diff
@@ -56,13 +56,16 @@ void exposeFrames() {
   bp::class_<FrameMotion>("FrameMotion",
                           "Frame motion describe using Pinocchio.\n\n"
                           "It defines a frame motion (tangent of SE(3) point) for a given frame ID",
-                          bp::init<FrameIndex, pinocchio::Motion>(bp::args("self", "frame", "oMf"),
-                                                                  "Initialize the frame motion.\n\n"
-                                                                  ":param frame: frame ID\n"
-                                                                  ":param oMf: Frame motion w.r.t. the origin"))
+                          bp::init<FrameIndex, pinocchio::Motion, bp::optional<pinocchio::ReferenceFrame> >(
+                              bp::args("self", "frame", "oMf", "reference"),
+                              "Initialize the frame motion.\n\n"
+                              ":param frame: frame ID\n"
+                              ":param oMf: Frame motion w.r.t. the origin\n"
```

Comment:
oMf is used in pinocchio to denote SE3 objects. It's not a good name for a motion reference

Reply:
If you are saving the reference frame in a separate object, it is better to not use the base index like o or i to denote the frame where the motion vector is defined.

Reply:
> oMf is used in pinocchio to denote SE3 objects. It's not a good name for a motion reference

Any suggested notation? I was though to use `omf` but it is hard to read.
As a side node, if we change this notation then we'll break the Python API.
So my question is "is this worth to do?"

> If you are saving the reference frame in a separate object, it is better to not use the base index like o or i to denote the frame where the motion vector is defined.

We don't save reference frame in other objects. The design choice is to be able to change reference frame as well.
Is this answer your question?

Reply:
Then I would suggest to deprecate this call and simply return the new motion object whenever this call is made. In the next release, we can finish the change of API.

> We don't save reference frame in other objects. The design choice is to be able to change reference frame as well.

What I mean is, Motion vector is defined in a base frame, and normally we write it like ov, or iv to say that it is denoted in frame o or i. But since you are saving a reference_frame as well inside this FrameMotion class, if you name this as ov, and then define the reference_frame to be LOCAL, that is conflicting. So I suggest simply to use a name without a base frame like o or i attached to it.

Reply:
Sorry the problem is not the constructor itself since it won't change the arguments (just their names). The problem is the member objects of the structure.

For instance, if we change `oMf` to `velocity` then we need to change the name of the member object. Let's my write a Python code to be more clear
```python
import crocoddyl
import pinocchio

frameId = 3
vref = crocoddyl.FrameMotion(frameId, pinocchio.Motion.Random(), pinocchio.LOCAL)
# This is the current names
print vref.oMf
# but the change of name in the constructor means to change it to
print vref.velocity
```

Reply:
I think it should be possible to duplicate `vref.velocity` under the name `vref.oMf`. Then if the user calls `vref.oMf`, he get the deprecated message and the value of `vref.velocity`